specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/2016-08-10/service.json

```diff
@@ -18,6 +18,46 @@
     "application/json"
   ],
   "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationVaultHealth": {
```

Comment:
No gaurav, This API will give give the health summary of immediate child resources only. At the vault level we are going with the exception which would give protected items level summary as well.
Yes the caller of the API will need to have the read access on the underlying resources and we will give the same. In the payload you can see each health summary contains the list of affectedResourceCorrelationId to related this health impacts which all resources.